app/src/main/java/com/nextcloud/talk/location/LocationPickerActivity.kt

```diff
@@ -2,7 +2,9 @@
  * Nextcloud Talk application
  *
  * @author Marcel Hibbe
+ * @author Ezhil Shanmugham
```

Comment:
no changes in this file, so no authorship, probably a copy&paste thing

Reply:
...or it is lacking the back-pressed changes

Reply:
I mistakenly added authorship while adding for the rest